mesoshttp/core.py

```diff
@@ -6,8 +6,10 @@ class CoreMesosObject(object):
     Internal class to manage driver
     '''
 
-    def __init__(self, mesos_url, frameworkId, streamId):
+    def __init__(self, mesos_url, frameworkId, streamId, requests_auth, verify):
```

Comment:
to keep compatibility, requests_auth and verify should be optional and have defaults

Reply:
Makes sense. I'll address this.